### PR TITLE
[fix][broker] change name limitTime to limitTimeInSec

### PR DESCRIPTION
**Pull request by StevenLuMT**
### Motivation

limitTime It's a confusing name, without the time unit, it is easy to cause bugs,
so we should deprecate the old name limitTime and introduce a new one limitTimeInSec

### Modifications
change name limitTime to limitTimeInSec

### Verifying this change
- [ ] Make sure that the change passes the CI checks.
This change is a trivial rework / code cleanup without any test coverage.

### Does this pull request potentially affect one of the following parts:

*If the box was checked, please highlight the changes*

- [ ] Dependencies (add or upgrade a dependency)
- [ ] The public API
- [ ] The schema
- [ ] The default values of configurations
- [ ] The threading model
- [ ] The binary protocol
- [ ] The REST endpoints
- [ ] The admin CLI options
- [ ] The metrics
- [ ] Anything that affects deployment

### Documentation
Check the box below or label this PR directly.

Need to update docs?

- [ ] `doc` 
- [ ] `doc-required` 
- [x] `doc-not-needed` 
- [ ] `doc-complete` 

### Matching PR in forked repository
PR in forked repository: https://github.com/StevenLuMT/pulsar/pull/1

### CODE REVIEW

**Review by congbobo184 — APPROVED** [review]

**Review by HQebupt — APPROVED** [review]
Good catch. LGTM

**Review by yuruguo — APPROVED** [review]

File: pulsar-client-admin-api/src/main/java/org/apache/pulsar/common/policies/data/impl/BacklogQuotaImpl.java [hunk 1]
```diff
@@ -80,12 +80,12 @@ public void setLimitSize(long limitSize) {
         this.limit = limitSize;
     }
 
-    public int getLimitTime() {
-        return limitTime;
+    public int getLimitTimeInSec() {
```
Comment (michaeljmarshall) [inline pulsar-client-admin-api/src/main/java/org/apache/pulsar/common/policies/data/impl/BacklogQuotaImpl.java:83]: This is a breaking change without any backwards compatibility.

**Review by michaeljmarshall — COMMENTED** [review]
@yuruguo @StevenLuMT - this is a breaking change without accounting for backwards compatibility. We either need to revert this or do something like https://github.com/apache/pulsar/pull/13291 (I haven't looked closely at that PR, but I think it is the right reference).

Note that this type of API change is technically supposed to go through the PIP process.

This type of change has far reaching implications for clients, too, so there must be new documentation for the change.

Let me know how you'd like to proceed.

File: pulsar-client-admin-api/src/main/java/org/apache/pulsar/common/policies/data/impl/BacklogQuotaImpl.java (hunk 1 above)
Comment (lhotari) [reply pulsar-client-admin-api/src/main/java/org/apache/pulsar/common/policies/data/impl/BacklogQuotaImpl.java:83]: I think it's better to revert this change and instead focus on improving documentation.

Comment (michaeljmarshall) [reply pulsar-client-admin-api/src/main/java/org/apache/pulsar/common/policies/data/impl/BacklogQuotaImpl.java:83]: I agree with that approach. There are many names in Pulsar that are slightly confusing. It'd make more sense to define a paradigm for naming before making many one-off changes.

**Comment by michaeljmarshall** [comment]
Reverting with #19152. I support working to improve the clarify of our names, but we need to be careful not to break any deployments.

**Comment by Jason918** [comment]
@StevenLuMT The right way is to keep compatibility, you can take the old `getLimit` for reference.
1. add getLimitTimeInSec in `BacklogQuota`
2. Deprecate getLimitTime in `BacklogQuota`
3. keep getLimitTime and setLimitTime  in `BacklogQuotaImpl`.

**Pull request merged**